python/its-status/its_status/main.py

```diff
@@ -6,53 +6,38 @@
 import argparse
 import configparser
 import its_status
```

Comment:
strange to import ... itself, no? Into my IDE, I've got a "No module named 'its_status' " so. Why to pass and mask through __init__?

Reply:
Sorry, I'm not sure to grok your last sentence. ;-)

How do you expect to be able to call the `init()` function that is defined
in the module, if not by importing the module?

As for why your IDE can't load the module, I'd guess that's an issue with
your IDE... Black reports no error, and it works flawlessly once the package
is installed. ;-p

Besides, that is not a change that was part of this MR, so if something really
needs to be done about it, that should be delegated to another MR (open an
issue, maybe?).

Reply:
It's more the fact to delegate important code (loop...)... into a __init__ file :(. i Would prefer to "just" "import" modules into this kind of file.

Reply:
Well, by having the init and loop in the module allows for that code to be reused in another client if need be.